src/main/java/com/resend/services/emails/Emails.java

```diff
@@ -62,7 +58,48 @@ public Email get(String emailId) throws ResendException {
 
             String responseBody = response.getBody();
 
-            Email email = resendMapper.readValue(responseBody, Email.class);
-            return email;
+        return resendMapper.readValue(responseBody, Email.class);
+    }
+
+    /**
+     * Update the schedule of an email by its unique identifier.
```

Comment:
```suggestion
     * Update the email by its unique identifier.
```

In the future this method might also update other attributes